test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -1973,6 +2001,7 @@ describe("USD Wallets - Lightning Pay", () => {
         recipientWalletId: walletIdA,
         senderAmountInvoice: usdSendAmount,
         recipientAmountInvoice: usdSendAmountInBtc,
+        txType: LedgerTransactionType.WalletIdTradeIntraAccount,
```

Comment:
do we need `WalletId`?

```suggestion
        txType: LedgerTransactionType.TradeIntraAccount,
```

Reply:
never mind, I understand now why we're doing that